decidim-core/app/forms/decidim/account_form.rb

```diff
@@ -28,6 +29,7 @@ class AccountForm < Form
     validates :password, password: { name: :name, email: :email, username: :nickname }, if: -> { password.present? }
     validates :password_confirmation, presence: true, if: :password_present
     validates :avatar, passthru: { to: Decidim::User }
+    validates :time_zone, time_zone: true
```

Comment:
```suggestion
    validates :time_zone, time_zone: true, if: -> { time_zone.present? }
```
We should validate this only if specified